### PR TITLE
Refactor Ordering

### LINKED ISSUE #28: Refactor \Cafe\UserInterface\Web\Controller\TabController::order

### PR DESCRIPTION
**Pull request by yourwebmaker**
closes #28

### CODE REVIEW

**Review by yourwebmaker — COMMENTED** [review]
@viniciusss or @zerai could you please review?

File: src/UserInterface/Web/Controller/TabController.php [hunk 1]
```diff
@@ -68,39 +63,18 @@ public function order(int $tableNumber, Request $request) : Response
...
-                for ($i = 0; $i < $item->numberToOrder; $i++) {
-                    $orderedItems[] = new OrderedItem(
-                        $item->menuNumber,
-                        $menu[$item->menuNumber]->description,
-                        $menu[$item->menuNumber]->isDrink,
-                        $menu[$item->menuNumber]->price,
-                    );
-                }
-            }
+        if ($request->isMethod(Request::METHOD_POST) ) {
```
Comment (vinyvicente) [inline src/UserInterface/Web/Controller/TabController.php]:
```suggestion
        if ($request->isMethod(Request::METHOD_POST)) {
```

File: composer.json [hunk 2]
```diff
@@ -18,7 +18,8 @@
         "symfony/framework-bundle": "5.1.*",
         "symfony/twig-bundle": "5.1.*",
         "symfony/yaml": "5.1.*",
-        "twig/intl-extra": "^3.1"
+        "twig/intl-extra": "^3.1",
+        "ext-json": "*"
```
Comment (vinyvicente) [inline composer.json]: why?

Comment (yourwebmaker) [reply composer.json]: The project has `JsonSerializable` usages and it's not as default there.

**Review by viniciusss — APPROVED** [review]

**Pull request merged**